app/code/Magento/Quote/Model/Cart/Totals/ItemConverter.php

```diff
@@ -71,7 +71,7 @@ public function __construct(
      * Converts a specified rate model to a shipping method data object.
      *
      * @param \Magento\Quote\Model\Quote\Item $item
-     * @return array
+     * @return \Magento\Quote\Model\Cart\Totals\Item
```

Comment:
I believe `\Magento\Quote\Api\Data\TotalsItemInterface` will be a better option here.

Reply:
Good point...I actually used that type in a plugin of this method!  See my new commit.